src/index.ts

```diff
@@ -1305,7 +1378,7 @@ export abstract class PinejsClientCore<PinejsClient> {
 			);
 		}
 		let { method, apiPrefix } = params;
-		const { body, passthrough = {} } = params;
+		const { body, passthrough = {}, retry = {} } = params;
```

Comment:
The default must be an object, not `false` because passing `false` would indicate "no retry".  If we default to `false`, then we effectively require the caller to always pass the retry parameters.

Reply:
If it defaults to an object that means it always retries instead.. I think the answer is actually not to default here at all but instead make `callWithRetry` allow undefined to be passed and have it handle any defaulting since it needs to be able to distinguish between `false`/`{}`/undefined